CONTRIBUTING.md

```diff
@@ -87,7 +87,7 @@ How to get your contributions merged smoothly and quickly:
 
 - The grpc package should only depend on standard Go packages and a small number
   of exceptions. **If your contribution introduces new dependencies**, you will
-  need a discussion with gRPC-Go maintainers. A github action check will run on
+  need a discussion with gRPC-Go maintainers. A Github action check will run on
```

Comment:
Apparently we have a special linter that will want this to be written as `GitHub` everywhere.

```suggestion
  need a discussion with gRPC-Go maintainers. A GitHub action check will run on
```